src/core/Toast/Toaster.tsx

```diff
@@ -10,6 +10,28 @@ import { ToastWrapper } from './ToastWrapper';
 
 const TOASTS_CONTAINER_ID = 'iui-toasts-container';
 
+export type ToasterSettings = {
+  /**
+   * Order of toasts.
+   * Ascending places toasts from newest to oldest - most recent toasts are on top. Descending - from oldest to newest - most recent toasts are on bottom.
+   * When placement is set and order not specified, toasts are ordered by placement. Top placement sets order 'descending', bottom placement sets order 'ascending'.
+   * @default 'descending'
```

Comment:
The `@default 'descending'` part is kinda wrong, since the default value is dynamic. So I'd say remove it and use the word "default" when you explain the behavior.

Something like:
```suggestion
   * When `placement` is set to a top value, order defaults to 'descending', otherwise 'ascending'.
```